project-base/storefront/components/Pages/ProductDetail/ProductDetailContent.tsx

```diff
@@ -41,33 +35,46 @@ export const ProductDetailContent: FC<ProductDetailContentProps> = ({ product, f
     useGtmProductDetailViewEvent(product, getUrlWithoutGetParameters(router.asPath), fetching);
 
     return (
-        // the key helps to re-mount the component when navigating between different products, which prevents the components from keeping an unwanted state
-        <Fragment key={product.uuid}>
```

Comment:
Do you know why we dont need it anymore?

Reply:
I have find no reason to keep it. I've also checked the scenario from the task which implemented this and I can't find any issues. https://shopsys.atlassian.net/browse/FWCC-1219

Reply:
Good, thanks. I have also tested it before and it seems it works. I just thought about asking you as you removed it 👍